code/modules/mob/living/simple_animal/abnormality/_tools/zayin/touch.dm

```diff
@@ -0,0 +1,23 @@
+/obj/structure/toolabnormality/touch
+	name = "don't touch me"
+	desc = "You feel like you shouldn't touch this."
+	icon_state = "touch"
+
+//I had to.
+/obj/structure/toolabnormality/touch/attack_hand(mob/living/carbon/human/user)
+	for(var/mob/M in GLOB.player_list)
+		if(M.z == z && M.client)
+			to_chat(M, "<span class='userdanger'>[user] WILL PUSH DON'T TOUCH ME.</span>")
+
+	if(do_after(user, 30 SECONDS))
+		SSticker.force_ending = 1
+		var/ending = pick(1,2)
+		switch(ending)
+			if(1)
+				for(var/mob/living/carbon/human/M in GLOB.player_list)
+					M.gib()
+			if(2)
+				for(var/mob/living/carbon/human/M in GLOB.player_list)
+					M.apply_damage(5000, WHITE_DAMAGE, null, null, spread_damage = TRUE)//You cannot escape.
```

Comment:
If we wanna make sure they can't un-panic set their white-resistance via physiology to 0.